themes/admin/Sea_Green/css/sea_green.css

```
@@ -18212,6 +18058,26 @@ ol.breadcrumb.title-bar-breadcrumb li.active {
+.box-widget .box-widget-card:hover {
+  outline: none;
+  border-radius: 4px;
+  border: 2px solid rgba(123, 97, 255, 0.4);
+}
+.box-widget .box-widget-card:focus-visible {
+  outline: none;
+  border-radius: 4px;
+  border: 2px solid rgba(123, 97, 255, 0.9);
+}
+.box-widget .box-widget-card:active {
+  outline: none;
```

Stylelint (17.11.0)

[error] Duplicate property "outline" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)